specifications/active/aRMT-1.4.3.yml

```diff
@@ -72,3 +72,11 @@ data:
     doc: Questionnaire application interaction event.
     topic: questionnaire_app_event
     value_schema: .monitor.questionnaire.QuestionnaireApplicationInteractionEvent
+  - type: ESM_DynaMORE
+    topic: questionnaire_ESM_DynaMORE
+    value_schema: .active.questionnaire.Questionnaire
+    questionnaire_definition_url: https://raw.githubusercontent.com/RADAR-base/RADAR-REDCap-aRMT-Definitions/master/questionnaires/ESM_DynaMORE/ESM_DynaMORE_armt.json
+  - type: ESM_DynaMORE_Evening
+    topic: questionnaire_ESM_DynaMORE_Evening
```

Comment:
As a convention can you please make the topic names all lowercase?

Reply:
Hi Yathar, of course. I updated both the 'type' and 'topic' fields in line with the conventional format :)